components/Accordion/Accordion.js

```diff
@@ -32,60 +28,72 @@ Accordion.defaultProps = {
   hasAnimationOnHover: false,
 };
 
+const slideDown = keyframes({
+  from: { height: 0 },
+  to: { height: 50 },
+});
+
+const slideUp = keyframes({
+  from: { height: 50 },
+  to: { height: 0 },
+});
+
+const StyledContent = styled(RadixAccordion.Content, {
+  overflow: 'hidden',
+
+  '&[data-state="open"]': {
+    animation: `${slideDown} 300ms cubic-bezier(0.87, 0, 0.13, 1)`,
+  },
+  '&[data-state="closed"]': {
+    animation: `${slideUp} 300ms cubic-bezier(0.87, 0, 0.13, 1)`,
+  },
+});
+
+const StyledChevron = styled(ChevronDownIcon, {
+  color: 'white',
+  width: 25,
+  height: 25,
+  transition: 'transform 300ms cubic-bezier(0.87, 0, 0.13, 1)',
+  '[data-state=open] &': { transform: 'rotate(180deg)' },
+});
+
 /**
  * @description A component whose main content is invisible until revealed by the user
  * @see http://web-accessibility.carnegiemuseums.org/code/accordions/
  */
 function Accordion({ accessibilityId, className, content, hasAnimationOnHover }) {
-  const [isContentVisible, setContentVisibility] = useState(false);
-
-  const toggleAccordionContent = () => setContentVisibility(previousState => !previousState);
-
   const contentId = `content-${accessibilityId}`;
   const accordionId = `accordion-control-${accessibilityId}`;
 
   return (
-    <Card
+    <RadixAccordion.Root
+      type="single"
       className={classNames(styles.Accordion, className)}
-      hasAnimationOnHover={hasAnimationOnHover}
+      collapsible
     >
-      <div className={styles.headingContainer}>{content.headingChildren}</div>
-
-      <button
-        aria-controls={contentId}
-        aria-expanded={isContentVisible}
-        className={styles.accordionToggleButton}
-        data-testid={ACCORDION_TOGGLE_BUTTON}
-        id={accordionId}
-        onClick={toggleAccordionContent}
-        type="button"
-      >
-        {isContentVisible ? (
-          <>
-            <ScreenReaderOnly>{toggleMessages.close}</ScreenReaderOnly>
-            <ChevronDown />
-          </>
-        ) : (
-          <>
-            <ScreenReaderOnly>{toggleMessages.open}</ScreenReaderOnly>
-            <ChevronRight />
-          </>
-        )}
-      </button>
-
-      <section
-        aria-hidden={!isContentVisible}
-        className={classNames(styles.accordionContent, {
-          [styles.visible]: isContentVisible,
-          [styles.invisible]: !isContentVisible,
-        })}
-        data-testid={ACCORDION_CONTENT}
-        id={contentId}
-        style={{ display: isContentVisible ? 'block' : 'none' }}
-      >
-        {content.bodyChildren}
-      </section>
-    </Card>
+      <RadixAccordion.Item value={accordionId}>
+        <RadixAccordion.Header>
+          <RadixAccordion.Trigger
+            className={classNames(styles.headingContainer, {
+              [styles.hover]: hasAnimationOnHover,
```

Comment:
Since you're no longer using our Card component, you dont need to expose `hasAnimationOnHover` as a prop to our accordion component, and you can simply always apply hover styles by changing the CSS to be:

```css
.headingContainer {
  /* styles */
}

.headingContainer:hover {
  cursor: pointer;
}
```

and then adjusting the className property value to be simply `styles.headingContainer`